fdbmonitor/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 set(FDBMONITOR_SRCS ConvertUTF.h SimpleIni.h fdbmonitor.cpp)
 
 add_executable(fdbmonitor ${FDBMONITOR_SRCS})
+assert_no_version_h(fdbmonitor)
```

Comment:
I'm not sure I understand why this line is necessary but other than that LGTM

Reply:
The way this is working is that it executes a custom target and makes every flow target a dependency of that. However, depending on the build system, fdbmonitor might be the first target that gets built and it will pick up versions.h. This makes sure that this build fails.

Reply:
Makes sense, thanks